TerminalOption desktop entry has been deprecated in the FreeDesktop specification.
http://standards.freedesktop.org/desktop-entry-spec/1.0/apc.html for details.
--- koffice-1.9.96.0~that.is.really.1.9.95.3.orig/filters/krita/ora/krita_ora.desktop
+++ koffice-1.9.96.0~that.is.really.1.9.95.3/filters/krita/ora/krita_ora.desktop
@@ -11,7 +11,6 @@
 Path=
 StartupNotify=true
 Terminal=false
-TerminalOptions=
 Type=Application
 X-DBUS-StartupType=multi
 X-DCOP-ServiceType=
--- koffice-1.9.96.0~that.is.really.1.9.95.3.orig/filters/krita/pdf/krita_pdf.desktop
+++ koffice-1.9.96.0~that.is.really.1.9.95.3/filters/krita/pdf/krita_pdf.desktop
@@ -10,7 +10,6 @@
 Path=
 StartupNotify=true
 Terminal=false
-TerminalOptions=
 Type=Application
 X-DBUS-StartupType=multi
 X-KDE-StartupNotify=true
--- koffice-1.9.96.0~that.is.really.1.9.95.3.orig/filters/krita/tiff/krita_tiff.desktop
+++ koffice-1.9.96.0~that.is.really.1.9.95.3/filters/krita/tiff/krita_tiff.desktop
@@ -11,7 +11,6 @@
 Path=
 StartupNotify=true
 Terminal=false
-TerminalOptions=
 Type=Application
 X-DBUS-StartupType=multi
 X-KDE-StartupNotify=true
